packages/kbn-i18n/src/core/i18n.ts

```diff
@@ -24,10 +24,17 @@ import IntlRelativeFormat from 'intl-relativeformat';
 import { Messages, PlainMessages } from '../messages';
 import { Formats, formats as EN_FORMATS } from './formats';
 import { hasValues, isObject, isString, mergeAll } from './helper';
+import { charMap } from './pseudo_locale';
 
 // Add all locale data to `IntlMessageFormat`.
 import './locales.js';
 
+/**
+ * Matches every single [A-Za-z] character, `<tag attr="any > text">` and `](markdown-link-address)`
+ */
+const CHARS_FOR_PSEUDO_LOCALIZATION_REGEX = /[A-Za-z]|(\]\([\s\S]*?\))|(<([^"<>]|("[^"]*?"))*?>)/g;
+const PSEUDO_LOCALE = 'en-xa';
```

Comment:
__issue:__ once moved, let's be more specific and call it `PSEUDO_ACCENTS_LOCALE`. I also see that it's usually used as `en-XA` (same as `en-US` and `ru-RU`), let's follow this rule unless you have objections.